src/components/Sidebar/contentTreeNode.js

```diff
@@ -122,25 +122,32 @@ const ContentTreeNode = ({ className, toggle, collapsed, url, title, location, c
       location.pathname === url + '/' ||
       location.pathname === config.metadata.pathPrefix + url);
   const collapse = () => {
+    console.log("test click", url);
```

Comment:
If this is for debugging purpose, could you please remove it?

Reply:
Sorry

Reply:
@mildronize No worries, nothing wrong.